pages/terra/policy.mdx

```diff
@@ -1,319 +0,0 @@
-    | System components | These packages simply cannot be sandboxed, as they are part of the operating system itself. |
-    | Libraries required to build or use the aforementioned packages | These libraries are required to build or use the aforementioned packages, and are usually not sandboxed. |
-    
-    Other applications that are not specified here may be allowed on a case-by-case basis.
-- You MUST have the rights or permission to redistribute the packaged software. Besides that, there is no prohibition on proprietary licenses.
-- Submissions must be maintained. We will reject submissions of unmaintained software.
-- Submissions MUST NOT be malicious.
-- Submissions that may introduce breaking changes must be included in "Terra Extras" instead via the `subrepo = "extras"` label in `anda.hcl`.
-  - Example: `terra-mesa` should be in extra as the dependency solver will resolve this package for matching "provides" (such as `mesa-libEGL(x86-64)`).
-  - Example: `libadwaita-nightly` should be in extra since it provides `pkgconfig(libadwaita-1)`, which is required by existing packages as a build dependency.
-  - Example: `yt-dlp-nightly` **can be included normally** because it has no common provides with Fedora's `yt-dlp`.
-  - If you are in doubt whether your submission should be in extra instead, ask and we will decide.
-- Submissions SHOULD include an auto-update script (`update.rhai`).
-- Submissions SHOULD be placed in the corresponding directory in `anda/`.
-  - Each folder MUST either contain folders and no source files (other files are exempted), or contain multiple files and no folders.
-  - The following directories have a higher priority. Categorize your new package in the following order:
-    1. `fonts/`
-    2. `system/` (bootloaders, kernels, drivers, ...)
-    3. `tools/buildsys/` (software used to build other software)
-    4. `devs/` (tools for software development)
-    5. `games/`
-    6. `themes/`
-    7. `docker/` (software related to containers)
-    8. `desktops/` (applications specific to a Desktop Environment)
-    9. `apps/` (GUI applications)
-    10. `langs/` (software written in/for a specific programming language)
-- When there are ambiguous or unspecified parts in our policies/guidelines, you SHOULD refer to [Fedora's packaging guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines).
-- Low-quality contributions will be declined at the team's discretion.
-- Do not include work-in-progress code or untested patches. See [dont-ship.it](https://archive.is/9zDrU).
-
-## Maintenance Policy
-
-- Anyone MAY make a PR (pull request) to an already existing package.
-- Unmaintained packages may be removed at any time. Before doing so, we will create a GitHub issue to gauge community interest.
-- Terra will only support the latest version of Fedora, if a package doesn't build on the next version of Fedora, we might hold it back.
```

Comment:
this is also missing,  you need to find somewhere to put the things in this doc or bring it back